Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -4064,7 +4052,7 @@ class CloneDeserializer : CloneBase {
 
     JSValue readBigInt()
     {
-        uint8_t sign = 0;
+        bool sign;
```

Comment:
Seems like this would not work if we are reading from a `m_version < 14` buffer. So if we do need the check in the other function, we probably need something different here.

Reply:
Ohhh, you're right. This was always encoded as a uint8_t so my read(bool&) will fail if `m_version < 14`. I'll address.